__tests__/helpers/daily-data-providers/combined-resting-heart-rate.test.ts

```
@@ -0,0 +1,128 @@
+        units: "",
+        startDate: startDate,
+        startDateOffset: "",
+        observationDate: startDate,
+        observationDateOffset: "",
+        insertedDate: "",
+        modifiedDate: "",
+        properties: { type: propertyType, day: startDate }
+    } as DeviceDataV2Point;
+    ddp.properties![propertyValueName] = value;
+    return ddp;
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)